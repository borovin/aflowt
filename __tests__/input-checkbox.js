import test from 'ava'
import Browser from './helpers/browser'
import {prettyPrint} from 'html'

test.beforeEach(t => {
  t.context.browser = new Browser()

  return t.context.browser.goto('/examples/input-checkbox.html')
})

test.afterEach(t => {
  return t.context.browser.close()
})

test('Checked checkbox', async t => {
  const snapshot = await t.context.browser.snapshot('b-input-checkbox:nth-child(1)')

  t.snapshot(prettyPrint(snapshot))
})

test('Unchecked checkbox', async t => {
  const snapshot = await t.context.browser.snapshot('b-input-checkbox:nth-child(2)')

  t.snapshot(prettyPrint(snapshot))
})