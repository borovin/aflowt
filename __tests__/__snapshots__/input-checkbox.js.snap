// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Checked checkbox 1`] = `
"<b-input-checkbox name=\\"f\\" checked=\\"\\" label=\\"checkbox\\">
    <label>
        <input name=\\"f\\" type=\\"checkbox\\" checked=\\"\\">
        <b-input-checkbox--icon>
            <svg viewBox=\\"0 0 24 24\\">
                <path d=\\"M9 16.17l-4.17-4.17-1.415 1.415 5.585 5.585 12-12-1.415-1.415z\\"></path>
            </svg>
        </b-input-checkbox--icon>
        <b-input-checkbox--text>checkbox</b-input-checkbox--text>
    </label>
</b-input-checkbox>"
`;

exports[`Unchecked checkbox 1`] = `
"<b-input-checkbox name=\\"g\\" label=\\"checkbox\\">
    <label>
        <input name=\\"g\\" type=\\"checkbox\\">
        <b-input-checkbox--icon>
            <svg viewBox=\\"0 0 24 24\\">
                <path d=\\"M9 16.17l-4.17-4.17-1.415 1.415 5.585 5.585 12-12-1.415-1.415z\\"></path>
            </svg>
        </b-input-checkbox--icon>
        <b-input-checkbox--text>checkbox</b-input-checkbox--text>
    </label>
</b-input-checkbox>"
`;
